viewer/src/index.ts

```diff
@@ -11,7 +11,7 @@ export { Color, Cognite3DViewerOptions, AddModelOptions } from './public/migrati
 export { CadLoadingHints } from './datamodels/cad/CadLoadingHints';
 export { BoundingBoxClipper } from './utilities';
 
-export { PotreePointColorType } from './datamodels/pointcloud/types';
+export { PotreePointColorType, PotreePointShape } from './datamodels/pointcloud/types';
```

Comment:
I'd suggest exporting everything to avoid some types being left not exported from the main entry.